tests/e2e/e2e_setup_test.go

```diff
@@ -24,7 +24,24 @@ import (
 	"github.com/osmosis-labs/osmosis/v7/tests/e2e/util"
 )
 
-const maxRetries = 10 // max retries for json unmarshalling
+var (
+	// common
+	maxRetries = 10 // max retries for json unmarshalling
+	// chainA
+	NumValA             int = 3
+	PruningA                = []string{"nothing", "default", "custom"} // default, nothing, everything, or custom
+	PruningKeepRecentA      = []string{"0", "0", "10000"}              // keep all of the last N states (only used with custom pruning)
+	PruningIntervalA        = []string{"0", "0", "13"}                 // delete old states from every Nth block (only used with custom pruning)
+	SnapshotIntervalA       = []uint64{1500, 1500, 1500}               // statesync snapshot every Nth block (0 to disable)
+	SnapshotKeepRecentA     = []uint32{2, 2, 2}                        // number of recent snapshots to keep and serve (0 to keep all)
```

Comment:
syntax is likely to be off but the idea is to have something like the following:
```suggestion
        validatorConfigsChainA = []*chain.ValidatorConfig{
             {
                 PruningStrategy: "default",
                 ....
             },
             {
                 PruningStrategy: "nothing",
                 ....
             },
             ...
        }
```